monitor/grafana/camunda-performance.json

```diff
@@ -1592,7 +1592,7 @@
                 "uid": "$DS_PROMETHEUS"
               },
               "editorMode": "code",
-              "expr": "histogram_quantile(0.99, sum(rate(zeebe_exporting_latency_bucket{namespace=~\"$namespace\", partition=~\"$partition\"}[$__rate_interval])) by (le))",
+              "expr": "histogram_quantile(0.99, sum(rate(zeebe_exporting_latency_bucket{namespace=~\"$namespace\", partition=~\"$partition\"}[$__rate_interval]) or rate(zeebe_exporting_latency_seconds_bucket{namespace=~\"$namespace\", partition=~\"$partition\"}[$__rate_interval])) by (le))",
```

Comment:
:question: is it correct that we don't need the `sum` in the second metric?

Reply:
`sum` is the outer call, the `or` is inside of it. So basically: `sum(rate(a) or rate(b))`, where `a` is the first series, and `b` is the new one.